packages/vaadin-login/test/login-form.test.js

```diff
@@ -151,6 +151,23 @@ describe('login form', () => {
     expect(submitSpy.called).to.be.false;
   });
 
+  it('should include CSRF in submit request if available', () => {
+    const loginWithCSRF = fixtureSync(
+      '<vaadin-login-form></vaadin-login-form><meta name="_csrf_parameter" content="_csrf" /><meta name="_csrf_header" content="X-CSRF-TOKEN" /><meta name="_csrf" content="28e4c684-fb5e-4c79-b8e2-a2177569edfa" />'
```

Comment:
Would it make sense to create `META` tags dynamically with JS and reuse a login form from a parent suite?

Reply:
Not sure it brings any additional value now that the test is in a separate suite